tests/test.env

```diff
@@ -6,6 +6,7 @@ WIS2NODE_DATADIR_PUBLIC=$WIS2NODE_DATADIR/data/public
 
 WIS2NODE_OSCAR_API_TOKEN=some_token
 WIS2NODE_URL=http://localhost/
+WIS2NODE_PIP3_EXTRA_PACKAGES="elasticsearch<8"
```

Comment:
Shouldn't this be for wis2node plugins?  If wis2node needs elastic, then this should be in `requirements.txt`.

Reply:
At the moment this is the "requirement" for the backend. This along with changes made is test.env, and wis2node.env are adding the very basic pipeline for that workflow. Probably should be inside requirements.txt seeing as elasticsearch is currently a dependency of wis2node for the backend.